runtime/src/qio/qbuffer.c

```diff
@@ -32,6 +32,7 @@
 #include <limits.h>
 #include <sys/mman.h>
 
+#include <ctype.h>
```

Comment:
ctype.h provides isprint which I use in debug_print_iovec. It's not needed in the qbuffer.h header file since it's just used in this function, not for any types for prototypes. We're getting iovec already for other reasons.